app/src/test/java/io/apicurio/registry/RegistryClientTest.java

```diff
@@ -107,7 +107,7 @@ void testSearchArtifact(RegistryRestClient restClient) throws Exception {
             Assertions.assertNotNull(artifactMetaData);
         });
 
-        ArtifactSearchResults results = restClient.searchArtifacts(name.toUpperCase(), SearchOver.name, SortOrder.asc, 0, 2);
+        ArtifactSearchResults results = restClient.searchArtifacts(name, SearchOver.name, SortOrder.asc, 0, 2);

```

Comment:
this shouldn't need to be changed, instead search query should be case insensitive

Reply:
Case insensitive searching for a SQL database is dependent on the configuration of the DB.  And it varies from DB to DB.  I think it's safer to not rely on case-insensitive searching in the unit test.

That said, I think you're probably right that searching should be case insensitive.  I'll open a GitHub issue for that and introduce a test (later) that is specific to that issue.

Reply:
ok thanks, I just was thinking that the new storage implementation needed some tweaks to do case-insensitive queries

Reply:
It might.  I'll work on that in a followup. :)  I opened a few followup GitHub issues.